lib/transformers/govukDateInput.js

```diff
@@ -22,6 +22,7 @@ module.exports = ({schemaKey, schema, options, data, schemaErrors} = {}) => {
                 {
                     label: 'Day',
                     classes: `govuk-input--width-2`,
+                    id: `${schemaKey}[day]`,
```

Comment:
This is to address [this](https://github.com/CriminalInjuriesCompensationAuthority/q-transformer/blob/master/lib/transformers/govukDateInput.js#L75) issue